2-mandatory/2-financial-times.js

```diff
@@ -14,68 +23,102 @@ function potentialHeadlines(allArticleTitles) {
     (you can assume words will always be seperated by a space)
 */
 function titleWithFewestWords(allArticleTitles) {
-    // TODO
+  let min = Infinity;
```

Comment:
Although it works, it seems a little weird to use `Infinity` (I've actually never used it in code). Maybe a better option would be to simply set the minimum to the length of the first one (you can set the index to 0 too) and then treat that one as the shortest. Then loop through and if any are shorter, replace it.

Reply:
Oh great idea thanks.